dominion-solutions/laravel
# Cu bcr2cn update laravel docker file ethan sowell

# Details

Added PHP7 libraries and the Pull Request Template

# Steps to test changes

1. Use `docker build . -f docker/app/Dockerfile` to build the image
1. Check there are no errors
1. Start the site with `docker-compose up`
1. Make sure it starts properly

# Issues Addressed

[Laravel-bcr2cn](https://app.clickup.com/t/bcr2cn)
